src/utils/filters.ts

```diff
@@ -0,0 +1,7 @@
+export default {
+    authorizedUrl(url) {
+        if (url && !url.includes("auth"))
+            return `${url}&auth=${import.meta.env.VITE_BMM_TOKEN}`;
```

Comment:
Since you've added the filter, please do it in a way which will also work when not having any query parameter (https://stackoverflow.com/a/44160941/517914)